sdk/networkanalytics/arm-networkanalytics/CHANGELOG.md

```diff
@@ -1,15 +0,0 @@
-# Release History
-
-## 1.0.1 (Unreleased)
```

Comment:
Could you use the hlc-code-gen change log tool?

Reply:
got the following error
![image](https://github.com/Azure/azure-sdk-for-js/assets/98569699/d8a8a725-5147-46bf-847e-0d7d55a3946e)